sql/src/main/java/com/vmware/ddlog/DDlogJooqProvider.java

```diff
@@ -343,6 +358,13 @@ private static String ddlogRelationName(final String tableName) {
         return "R" + tableName.toLowerCase();
     }
 
+    /*
+     * This corresponds to the naming convention followed by the SQL -> DDlog compiler
+     */
+    private static String relationNameToTableName(final String relationName) {
```

Comment:
I think these should become methods in the compiler.

Reply:
In the sql-ddlog compiler right? I'll send a separate PR for that.